plugins/rest/rest_handlers.go

```diff
@@ -247,9 +247,9 @@ func (plugin *Plugin) registerHTTPHandler(key, method string, f func() (interfac
 
 			res, err := f()
 			if err != nil {
-				plugin.Deps.Log.Errorf("Error: %v", err)
-				errStr := fmt.Sprintf("500 Internal server error: %s\n", err.Error())
-				formatter.Text(w, http.StatusInternalServerError, errStr)
+				errMsg := fmt.Sprintf("500 Internal server error: request failed: %v\n", err)
+				plugin.Log.Error(errMsg)
+				formatter.Text(w, http.StatusInternalServerError, errMsg)
```

Comment:
This seems to be inconsistent with the other errors.. returning text vs json.

Reply:
Didn't notice, thanks.